.github/workflows/test.yml

```diff
@@ -66,15 +68,23 @@ jobs:
           -r tests/requirements.txt
           .
 
-      - name: Test
+      - name: Test (Linux/MacOS)
+        if: runner.os != 'Windows'
+        run: python -m pytest -v -m ctest
+        env:
+          # Set the PYTHONPATH to the downloaded-artifact
+          # Even though we are running C++ tests, pytest still discovers
+          # all the python files, so they need to be valid imports.
+          PYTHONPATH: ./openassetio/lib/python3.9/site-packages
+          OPENASSETIO_TRAITGENTEST_CMAKE_PRESET: lint
+
+      # The site-packages layout on windows is different.
+      - name: Test (Windows)
+        if: runner.os == 'Windows'
         run: python -m pytest -v -m ctest
         env:
-          # Set the PYTHONPATH to the output of build-openassetio
+          # Set the PYTHONPATH to the downloaded-artifact
           # Even though we are running C++ tests, pytest still discovers
           # all the python files, so they need to be valid imports.
-          PYTHONPATH: ${{ github.workspace }}/openassetio/lib/python3.9/site-packages
-          CMAKE_PREFIX_PATH: ${{ github.workspace }}/openassetio
-          # TODO(DF): Sanitizers disabled due to lack of support in ASWF
-          # Docker image. Re-enable once per-platform builds enabled
-          # (i.e. once OpenAssetIO Conan package is available).
-          OPENASSETIO_TRAITGENTEST_CMAKE_PRESET: ci
+          PYTHONPATH: ./openassetio/lib/site-packages
+          OPENASSETIO_TRAITGENTEST_CMAKE_PRESET: lint
```

Comment:
[Question] Whats the deal with the change of cmake preset and was it intended to remove that comment?

Reply:
Yes. Changing the cmake preset re-enables the sanitizers. This is fulfilling the todo.

Reply:
Thinking about it, was the original intent of the `ci

Reply:
Removed the ci preset entirely : https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/56/commits/d961f40262723fb8a462891c91e2ea78063460a9